### DISCUSSION TITLE
[Bug]: Changing Port

### DISCUSSION THREAD

**Discussion by FrancyMak** [coder/code-server#5456] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS/Web Information

- Web Browser: Opera
- Local OS: Windows 10
- Remote OS: Ubuntu
- Remote Architecture: amd64
- `code-server --version`: 4.5.1

### Steps to Reproduce

Good evening, I successfully managed to install code-server on my dedicated Ubuntu 22.04 LTS on port 8080 which I left free only for the first boot of code-server.

I followed the commands in your documentation for changing the port, but every time it resets and returns to 8080 even though I start it this way `code-server --bind-addr localhost:3000` or this way `PORT=3000 code-server` or even editing the config file (where the password is saved). How can I make this change permanent?

### Expected

It should start on the selected port

### Actual

It always starts on the 8080 creating problems for other connections.

### Does this issue happen in VS Code or GitHub Codespaces?

- [X] I cannot reproduce this in VS Code.
- [X] I cannot reproduce this in GitHub Codespaces.

### Are you accessing code-server over HTTPS?

- [X] I am using HTTPS.

**Comment by balajib-b** [comment, marked as answer]
$ nano ~/.config/code-server/config.yaml
Add this line: bind-addr: 127.0.0.1:3000
